.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby
+
+on:
+  [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['2.6', '2.7'] #repo doesn't yet support 3.0 :upside_down_face:
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+    - name: Setup Test Env
+      run: |
+        touch .env
+        echo BT_ENVIRONMENT='sandbox' >> .env
+        echo BT_MERCHANT_ID='${{ secrets.BT_MERCHANT_ID }}' >> .env
+        echo BT_PUBLIC_KEY='${{ secrets.BT_PUBLIC_KEY }}' >> .env
+        echo BT_PRIVATE_KEY='${{ secrets.BT_PRIVATE_KEY }}' >> .env
```

Comment:
This piece _shouldn't_ be necessary. The way `dotenv` packages typically work is that they pull in environmental variables from an .env file, as well as the ones set at the system level. I'd move the `BT_ENVIRONMENT` to a secret in the repo, and eliminate this section. (assuming there's nothing wonky going on with the `dotenv` package)


```suggestion
```